src/commands/yarn_install.yml

```diff
@@ -9,6 +9,9 @@ parameters:
     description: The path to the yarn cache folder.  Defaults to /tmp/yarn
     type: string
     default: "/tmp/yarn"
+  yarn_install_directory:
+    description: The working directory to run install at. Defaults to currenct directory
+    type: string
```

Comment:
Did you forget to add a `default:` field?

Reply:
Oops! You are right 😅 First day back from holiday stuff 🤷‍♂️ 

Fixed in new commit!